modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -67,39 +68,47 @@ If you want to create virtual function on the default network interface, ensure
 
 <8> The number of the virtual functions (VF) to create for the SR-IOV physical network device. For an Intel network interface controller (NIC), the number of VFs cannot be larger than the total VFs supported by the device. For a Mellanox NIC, the number of VFs cannot be larger than `128`.
 
-<9> The NIC selector identifies the device for the Operator to configure. You do not have to specify values for all the parameters. It is recommended to identify the network device with enough precision to avoid selecting a device unintentionally.
+<9> Set `externallyManaged` to `true` to allow the SR-IOV Network Operator to use all or a subset of externally managed virtual functions (VFs) and attach them to pods. With the value set to `false` the SR-IOV Network Operator manages and configures all allocated VFs.
++
+[NOTE]
+====
+When `externallyManaged` is set to `true`, you must create the Virtual Functions (VFs) before applying the policy. If not, the webhook will block the request.
+If `externallyManaged` is set to `false`, the SR-IOV Network Operator handles the creation and management of VFs, including resetting them if necessary. Therefore to use VFs on the host system they must be created manually and `externallyManaged` must be set to `true` so the SR-IOV Operator Network will not take any actions on the created VFs.
```

Comment:
so the SR-IOV Network Operator will not take any actions on the PF  and the VFs that are not in the policy nitSelector.

Reply:
Hi @SchSeba is this feature now released. I was waiting for the z-stream. Is it out and in what z-stream ?

Reply:
HI @SchSeba should this read "so the SR-IOV Network Operator will not take any actions on the PF and the VFs that are not in the policy nodeSelector" ?

Reply:
@SchSeba got it I think "so the SR-IOV Network Operator will not take any actions on the PF and the VFs that are not defined in the policy nicSelector"

Reply:
yep I think it sounds good thanks!

the backport to 4.14.z is WIP should be done this week